api-tests/api.test.js

```diff
@@ -89,3 +102,93 @@ test('/events endpoint', async () => {
     ])
   })
 })
+
+test('/events/{eventKey}/info endpoint', async () => {
+  const d = await fetch(addr + '/events/2018flor/info').then(d => d.json())
+  const info = d.data
+  expect(info.key).toEqual('2018flor')
+  expect(info.name).toBeA(String)
+  expect(info.startDate).toBeADateString()
+  expect(info.endDate).toBeADateString()
+  expect(info.location).toBeA(Object)
+  expect(info.location.name).toBeA(String)
+  expect(info.location.lat).toBeA(Number)
+  expect(info.location.lon).toBeA(Number)
+  expect(info.key).toBeA(String)
+  expect(info.district).toBeUndefinedOr(String)
+  expect(info.week).toBeUndefinedOr(Number)
+  expect(info.webcasts).toEqual(expect.any(Array))
+  info.webcasts.forEach(webcast => {
+    expect(webcast.url).toBeA(String)
+    expect(webcast.type).toBeA(String)
+    expect(webcast.type).toEqual(expect.stringMatching(youtubeOrTwitch))
+    expect(Object.keys(webcast)).toBeASubsetOf(['type', 'url'])
```

Comment:
I think you could use

```js
expect(webcast).toEqual({
  url: expect.any(String),
  type: expect.stringMatching(youtubeOrTwitch)
})
```

here since `webcast` does not have any optional keys